src/main/steps/home/get.ts

```diff
@@ -32,8 +33,10 @@ export class HomeGetController {
       );
     }
 
-    if (req.session.userCase.state === State.Applicant2Approved) {
-      res.redirect(`${CONFIRM_JOINT_APPLICATION}`);
+    if (req.session.userCase.applicant2ScreenHasUnionBroken === YesOrNo.NO) {
```

Comment:
i feel like this should be an event which gets checked. what if the user clicked no but didnt end the case?

Reply:
@velimd yeah I see what you're saying, do you think it would be better to have a state change when applicant 2 ends the application on the 'you-cannot-apply' page? 
Or maybe add a field to check if applicant2 has ended the application?

@linusnorton

Reply:
I guess we could set it to AwaitingApplicant1 and then also check for this field